licenses.d2iq.yaml

```diff
@@ -3,7 +3,7 @@ ignore:
   - ghcr.io/mesosphere/dkp-container-images/docker.io/mesosphere/grafana-plugins:v0.0.1-d2iq.0
   - docker.io/mesosphere/kommander2-kubetools
   - docker.io/nginxinc/nginx-unprivileged:1.24.0-alpine
-  - docker.io/bitnami/external-dns:0.14.1-debian-12-r2
+  - docker.io/bitnami/external-dns:0.14.2-debian-12-r1
```

Comment:
@SandhyaRavi2403 external-dns is a special case here, if you looked at the failing licenses check, the error was `Error: No mapping for image "docker.io/bitnami/external-dns:0.14.2-debian-12-r1"`. This is because we specify the external-dns image in this `ignore` block, and you will need to update this image manually, our automation can't handle this yet. (I've pushed this change to show you)